docusaurus/docs/core-concepts/config-store/overview.md

```diff
@@ -1,35 +1,56 @@
 
 ## Overview
-The configuration store has four components:
 
-1. **Configuration types** 
-    1. Configuration types define a type of configuration, including an optional JSON schema that the configuration data must conform to.
-    1. Configuration types have the following attributes: 
-        1. A name 
+The configuration store has several components:
+
+1. **Config Types**
+    1. Config Types define a type of Config, including an optional JSON schema that the configuration data must conform to.
+    1. Config Types have the following properties:
+        1. A Config Type name
         1. An optional JSON schema to validate configurations of the type
-        1. Standard edge attributes: id, tags, createdAt, updatedAt
-1. **Configurations** 
-    1. Configurations have the following attributes: 
-        1. A name
+        1. Standard edge entity properties: id, tags, createdAt, updatedAt
+1. **Configs**
+    1. Configs have the following properties:
+        1. A Config name
         1. The configuration data, which is arbitrary JSON data, so long as it conforms to the type schema (if specified)
-       1. Standard edge attributes: id, tags, createdAt, updatedAt
-1. **Services** 
-    1. Each service can be linked to multiple configuration. Services can have one configuration linked for each configuration type.
-1. **Identities** 
-    1. An identity can have a configuration specified for a given service and configuration type. This will override any configuration for the service for that type 
+        1. Standard edge entity properties: id, tags, createdAt, updatedAt
+1. **Services**
+    1. Each service can be linked to multiple Configs. Services can have one linked Config for each Config Type.
+1. **Identities**
+    1. An identity may have a Config specified for a given service and Config. This will override the service's linked Configs.
 
 This configuration model has the following properties:
-* Different applications can have their own configuration for the same service
-* Applications can have multiple configuration types for themselves where it makes sense
-    * Ziti tunnelers have one type for the client side and one for the server side, since they have different properties and not every service with use both
-* Since an application can support multiple configuration types, applications can version their configuration types as their needs change
+
+* Different applications can have their own Configs for the same service
+* Applications can have multiple Config Types for themselves where it makes sense
+* Ziti tunnelers use standard Config Types for intercept (client) side and hosting (server) side
+* Since an application can support multiple Config Types, applications can version their Config Types as their needs change
+
+## Standard Config Types
```

Comment:
btw this section gets into territory that is more about "tunnelers" than "config types" imo, and I think it'd fit nicely into tunneler docs once we get that domain organized.

in the meantime, I'd prefer "Tunneler" to "Standard" in the section heading.

Reply:
agreed